test/extended/builds/start.go

```diff
@@ -409,6 +411,20 @@ var _ = g.Describe("[sig-builds][Feature:Builds][Slow] starting a build using CL
 			})
 
 			g.Describe("start a build via a webhook", func() {
+
+				// AUTH-509: Webhooks do not allow unauthenticated requests by default.
+				// Create a role binding which allows unauthenticated webhooks.
+				g.BeforeEach(func() {
+					ctx := context.Background()
+					_, err := oc.AdminAuthorizationClient().AuthorizationV1().RoleBindings(oc.Namespace()).Get(ctx, "webooks-unauth", metav1.GetOptions{})
+					if apierrors.IsNotFound(err) {
+						err = oc.AsAdmin().Run("apply").Args("-f", webhookRoleBinding).Execute()
```

Comment:
use the client to create, not an `oc` forked process.

Reply:
My intent here was to simulate admin/end user behavior that we are going to recommend in our docs. Is running `oc` in a forked process now discouraged?

Reply:
> My intent here was to simulate admin/end user behavior that we are going to recommend in our docs. Is running `oc` in a forked process now discouraged?

Yes.  It has proven difficult to debug.  So for the build features, I'm +1 to using the `oc` process.  but for this "add a permission" I'd like to use a normal client

Reply:
ack.

As I was updating, I found I was technically using the "wrong client" to check for the rolebinding. Fixed to use the `AdminKubeClient`.